src/firefly/js/ui/tap/ObjectIDSearch.jsx

```diff
@@ -175,14 +224,14 @@ function UploadTableSelectorObjectID({uploadInfo, setUploadInfo}) {
                 headerStyle:{paddingLeft:1},
                 style:{margin:'0 0 10px 195px'},
                 labelStyle:{paddingRight:10},
-                objectKey:UploadObjectIDColumn }} />
+                objectKey:UploadObjectIDColumn}} />
             }
         </div>
     );
 }
 
 function ObjectIDCol({objectCol, style={},cols, objectKey, openKey, labelStyle,
-                           headerTitle, headerPostTitle = '', openPreMessage='', headerStyle}) {
+                           headerTitle, headerPostTitle = '', openPreMessage='', headerStyle,tbl_id=null,clickingSelectCols=null,setClickingSelectCols=null}) {
```

Comment:
It's a lot cleaner if you just move the useState/useEffect to this component.  Passing these value/function as props to its children is messy especially when it's not needed.